### ISSUE TITLE
The loss does not decrease when I try to use RBF-SOFTMAX  in my own code.

### ISSUE THREAD

**Issue by Hongbin98** [2han9x1a0release/RBF-Softmax#1]
Hello, my question is: when I copy the "rbflogit.py" to my project and it is used to replace the "nn.Linear()", I get the right logits but the loss does not decrease. At the same time, I replace the "RBFLogits()" to "LinearLogits" and it works. So I wonder if there is any question in the optimization stage of the "RBFLogits()".
Looking forward to your reply!

**Comment by Hongbin98** [comment]
I am a newbie in this field, so please feel free to advise~

**Comment by 2han9x1a0release** [comment]
Thank you for using RBF-softmax. Here are several suggestions:
1. Trying to enlarge scale hyperparameter first; or
2. Using more iterations/epochs for training, because some time RBF loss values changes a little in early stage; or
3. Maybe you can give a brief description of your dataset as well as backbone. So that maybe I can give you more detailed suggestions.

**Comment by 392398741nm** [comment]
Hello, I replace "nn.Linear"  to "rbflogit.py" on MNIST. But the accuracy of the training set and the test set is always 0.1. Epoch was set to 80 and scale hyperparameter has been larged. If I use other loss functions, the net will work normally. I don't know why.  Looking forward to your reply!
